runtime/src/bank.rs

```diff
@@ -3648,7 +3635,12 @@ impl Bank {
                         }
                     }
                 }
+            } else if *feature_id == feature_set::cross_program_invocation_enabled::id()
```

Comment:
@mvines What do you think about this as a way to consolidate existing epoch dependent features into FeatureSet? Doing it this way means the feature would turn on at some epoch or we could enable it at anytime before.

Reply:
Is this just a transition problem because CPI not fully rolled out on all clusters?   Once we're fully migrated over I feel like these special cases go away. I ran into a similar case at 5d6410c1cb662a31b4c5bb04efced666d7eff29a and solved it a little differently.

The problem I see with adding that info into `FeatureSet` is that then the `Feature` account won't be setup correctly (without further Bank integration), so `solana feature status` won't reflect the actual status of that feature.

In the case of `consistent_recent_blockhashes_sysvar`, I still need to `solana feature activate` this feature on devnet/testnet even though it's already actually active.  This is just for accounting purposes to ensure the `Feature` account is configured correctly at the next epoch on devnet/testnet.

Reply:
@mvines CPI (at least this feature) is rolled out on all the clusters already, on test/devnet its on by default and on mainnet is after epoch 63.  Maybe we don't even need this feature anymore, just always on?

I want this feature in FeatureSet so it's available to the runtime without threading a special case bool.  Looks like the way you solved `consistent_recent_blockhashes_sysvar` is very similar to this given the feature may be activated via account but also activated without an account existing.  So both solutions an account can be created to satisfy the `solana feature status`.

Reply:
> Maybe we don't even need this feature anymore, just always on?

Oh yeah, if it's already enabled on all three clusters then I think the feature can be removed from the code entirely.